src/collections/Grid/GridColumn.js

```diff
@@ -88,6 +90,9 @@ GridColumn.propTypes = {
   /** A column can appear only for a specific device, or screen sizes. */
   only: PropTypes.oneOf(GridColumn._meta.props.only),
 
+  /** An can stretch its contents to take up the entire column height. */
```

Comment:
Does the column take up the "entire row height" as well? Instead of entire column height.